10-setup-htcondor.sh

```diff
@@ -228,23 +219,64 @@ EOF
 
 fi
 
-if is_true "$ITBPOOL"; then
-    OSPOOL_CM1=cm-1.ospool-itb.osg-htc.org
-    OSPOOL_CM2=cm-2.ospool-itb.osg-htc.org
+
+######################
+# POOL CONFIGURATION #
+######################
+
+# Default to the production OSPool
+POOL=${POOL:=prod-ospool}
+
+case ${POOL} in
+    itb-ospool)
+        CCB_RANGE_LOW=1
+        CCB_RANGE_HIGH=5
```

Comment:
Where did `CCB_RANGE_LOW` and `CCB_RANGE_HIGH` come from before this change?

Reply:
We didn't set it before per se but the values came from https://github.com/opensciencegrid/osgvo-docker-pilot/pull/63/files#diff-8dbb461a6753f09dffe3e11c79e48c6e7b194d2392514fd4024b7fbbbd435b5aL245